keepassxc-browser/_locales/en/messages.json

```diff
@@ -0,0 +1,742 @@
+        "description": "A popup message shown when the extension has not been connected to KeePassXC."
+    },
+    "popupNeedReconfigure": {
+        "message": "KeePassXC-Browser has been disconnected from KeePassXC.",
+        "description": "A popup message shown when the extension has been disconnected from KeePassXC."
+    },
+    "popupNeedReconfigureMessage": {
+        "message": "Press the reconnect button to establish a new connection.",
+        "description": "A popup message shown when reconnect is needed."
+    },
+    "popupConfiguredNotAssociated": {
+        "message": "KeePassXC-Browser has not yet connected to KeePassXC but has been configured using the identifier: $1",
+        "description": "A popup message shown when KeePassXC is not connected but configured."
+    },
+    "popupConfiguredAndAssociated": {
+        "message": "KeePassXC-Browser is successfully connected to KeePassXC and has been configured using the following identifier: $1",
+        "description": "A popuo message when a connection to KeePassXC is succesful."
+    },
+    "popupRememberInfoText": {
+        "message": "Username or password changed! Save it?",
+        "description": "Popup message when username or password has changed."
+    },
+    "popupRememberSaving": {
+        "message": "Credentials will be saved in connected database with identifier $1",
+        "description": "A popup message when saving or updating credentials."
+    },
+    "popupRememberNewUsername": {
+        "message": "The following username used is currently not saved: $1",
+        "description": "A popup message shown when a new username is detected when saving credentials."
+    },
+    "popupRememberUsernameExists": {
+        "message": "The credentials with the used username are marked bold: $1",
+        "description": "A popup message shown when a existing username is detected when saving credentials."
+    },
+    "popupRememberChooseCredentials": {
+        "message": "Please choose the credentials you want to update.",
+        "description": "A popup message shown choosing what credentials user wants to update."
+    },
+    "popupLoginText": {
+        "message": "Select the login information you would like to get entered into the page.",
+        "description": "A popup message shown when one or multiple credentials are present."
+    },
+    "popupFilterText": {
+        "message": "Filter:",
+        "description": "Text for login credentials quick filter."
+    },
+    "popupAuthText": {
+        "message": "Select the login information you would like to get logged in with.",
+        "description": "A popup message shown when doing HTTP Authentication with multiple credentials present."
+    },
+    "popupMultiplePasswordFields": {
+        "message": "KeePassXC-Browser found more than one password field on this page. To enter your logins, right-click on one of the password fields, and choose either of the commands:",
```

Comment:
More than one password field found on this page. Right-click one of the password fields, and choose one of:

(again, this might be a candidate for placeholders instead of string concatenation)

Reply:
After this message a credential list is shown where the user can choose on of them. So in this case the placeholder won't work.